app/static/admin/css/forms.css

```diff
@@ -516,4 +516,10 @@ form .related-widget-wrapper ul {
 
 #donation_result_list tr:hover, #item_result_list tr:hover{
     cursor: pointer;
+}
+
+.ui-autocomplete {
+    height: 200px;
+    overflow-y: scroll;
+    overflow-x: hidden;
```

Comment:
Probably should do overflow-x: auto since there may be cases where names are long.

Reply:
Sure.